docs/client_quickstart.rst

```diff
@@ -55,6 +55,18 @@ Other HTTP methods are available as well::
     session.options('http://httpbin.org/get')
     session.patch('http://httpbin.org/patch', data=b'data')
 
+To make several requests to the same site more simple, the parameter ``base_url``
+of :class:`ClientSession` constructor can be used. For example to request different
+endpoints of ``http://httpbin.org`` can be used the following code::
+
+    async with aiohttp.ClientSession('http://httpbin.org') as session:
+        async with session.get('/get'):
```

Comment:
@asvetlov 
It works as expected by RFC, but I think the feature wasn't about RFC implementation.
May be it would be better if `base_url='http://httpbin.org/status'` and `url='/200'` build `http://httpbin.org/status'/200`.
Not RFC complaint, but I would expect such behavior.

Reply:
Another option is following the current `url / path` behavior: 
`base_url='http://httpbin.org/status'` and `url='/200'` is forbidden
`base_url='http://httpbin.org/status'` and `url='200'` builds `http://httpbin.org/status/200`

BTW, should we forbid query/fragment parts in `base_url`. They are ignored anyway.

Reply:
Let's wait for the user feedback on it.